internal/repository/root.go

```diff
@@ -60,37 +73,47 @@ func (r *Repository) AddTopLevelTargetsKey(ctx context.Context, rootKeyBytes, ta
 	if err != nil {
 		return err
 	}
+	slog.Debug("Loaded new signer verifier in custom ssl format to generate key id of root of trust")
 	rootKeyID, err := sv.KeyID()
 	if err != nil {
 		return err
 	}
+	slog.Debug("Created root key id with signer verifier")
 
 	state, err := policy.LoadCurrentState(ctx, r.r)
 	if err != nil {
 		return err
 	}
+	slog.Debug("Loaded current state of repository's active policy")
 
 	rootMetadata, err := state.GetRootMetadata()
 	if err != nil {
 		return err
 	}
+	slog.Debug("Constructed root meta data with deserialized paylod of state's root envelope")
 
 	if !isKeyAuthorized(rootMetadata.Roles[policy.RootRoleName].KeyIDs, rootKeyID) {
 		return ErrUnauthorizedKey
 	}
+	slog.Debug("Successfully authorized root of trust with root key id")
 
 	targetsKey, err := tuf.LoadKeyFromBytes(targetsKeyBytes)
 	if err != nil {
 		return err
 	}
+	slog.Debug("Loaded pointer to the key generated from targets key bytes")
```

Comment:
```suggestion
	slog.Debug("Loaded policy key")
```

I'd again add in the key ID for this one as well.

General tip: the log statement does not need to contain extremely granular details about the _implementation_ (eg: pointer to the key does not matter to a gittuf user). Rather, we want to log important activities as they happen with pertinent information like the key IDs, things that users will care about.

Reply:
I think statements can be more brief as well.